src/Tasks.UnitTests/ResourceHandling/GenerateResource_Tests.cs

```diff
@@ -3711,16 +3724,27 @@ public static string WriteTestResX(bool useType, string linkedBitmap, string ext
         /// <param name="useType">Indicates whether to include an enum to test type-specific resource encoding with assembly references</param>
         /// <param name="linkedBitmap">The name of a linked-in bitmap.  use 'null' for no bitmap.</param>
         /// <returns>The name of the resx file</returns>
-        public static string WriteTestResX(bool useType, string linkedBitmap, string extraToken, bool useInvalidType, string resxFileToWrite = null)
+        public static string WriteTestResX(bool useType, string linkedBitmap, string extraToken, bool useInvalidType, string resxFileToWrite = null, TestEnvironment env = null)
         {
             string resgenFile = resxFileToWrite;
-            if (string.IsNullOrEmpty(resgenFile))
+
+            string contents = GetTestResXContent(useType, linkedBitmap, extraToken, useInvalidType);
+
+            if (env == null)
+            {
+                if (string.IsNullOrEmpty(resgenFile))
+                {
+                        resgenFile = GetTempFileName(".resx");
+                        File.Delete(resgenFile);
```

Comment:
(nit) Is this line necessary? According to the [File.WriteAllText documentation](https://docs.microsoft.com/en-us/dotnet/api/system.io.file.writealltext?view=netframework-4.8), even if the file did exist it would be overwritten

Reply:
It's preexisting but no, I don't think it is. I'll remove.